web/src/test/java/com/epam/esm/controller/TagControllerTest.java

```diff
@@ -0,0 +1,104 @@
+package com.epam.esm.controller;
+
+import com.epam.esm.dto.TagDto;
+import com.epam.esm.service.TagService;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class TagControllerTest {
+
+    @Mock
+    private TagService serviceMock;
+
+    @InjectMocks
+    private TagController tagController;
+    private static TagDto defaultTagDto;
```

Comment:
@gkttk Why this variables are `static`?

Reply:
State of the variable defaultTagDto never changes so i uses it in @BeforeAll code block which  must be a static method that has access only to static fields.